src/DripsHub.sol

```diff
@@ -94,37 +94,37 @@ abstract contract DripsHub {
     );
 
     /// @notice Emitted when the user's splits are updated.
-    /// @param user The user
+    /// @param userId The user ID
     /// @param receivers The list of the user's splits receivers.
-    event SplitsUpdated(uint256 indexed user, SplitsReceiver[] receivers);
+    event SplitsUpdated(uint256 indexed userId, SplitsReceiver[] receivers);
```

Comment:
Do you think we should have the `accountId` indexed as well?

I don't think so. We would say, use the events from the ERC721 if you want to see all tokenHolder drips?

Is this a correct assumption?

Reply:
Yeah, I think so too. The DripsHub events should be agnostic about the meaning of the userId, it should be just a number. If you know how to interpret it, good for you, but it's not what DripsHub should focus on and make its users pay for.